src/app/shared/components/checkout/cost-center-selection/cost-center-selection.component.html

```diff
@@ -0,0 +1,6 @@
+<ng-container *ngIf="showSelection">
```

Comment:
(costCenterOptions$ | async)?.length && isBusinessCustumer | async

something like this